bigtable/client/internal/readrowsparser.h

```diff
@@ -68,15 +68,16 @@ class ReadRowsParser {
    * @throws std::runtime_error if validation failed.
    */
   virtual void HandleChunk(
-      google::bigtable::v2::ReadRowsResponse_CellChunk chunk);
+      google::bigtable::v2::ReadRowsResponse_CellChunk chunk,
+      grpc::Status &status);
```

Comment:
I am sure `clang-format(1)` will tell you this, but we are using `grpc::Status& status` (the `&` is aligned to the left).